### PR TITLE
Add faq

### LINKED ISSUE #2028: Add note on potential capa FP detection to documentation
Actually, before the issue is closed, is there a false positive reporting process for Windows Defender and the other three engines that can be pursued to prevent this causing trouble for new users, or is this expected enough that it can be put perhaps into an FAQ for the program on this github project?

_Originally posted by @RionEV in https://github.com/mandiant/capa/issues/2025#issuecomment-1988802299_

### PR DESCRIPTION
**Pull request by mr-tz**
closes #2028 

cc: @RionEV

### Checklist

- [ ] No CHANGELOG update needed

- [x] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
+## Why does capa trigger my Antivirus? Is the tool safe to use?
+capa's purpose is to analyse the capabilities of a potentially malicious application or file. In order to do that, it needs to contain parts of the data it's designed to detect as a basis of comparison.
```
Comment (VascoSch92) [inline doc/faq.md:3]:
```suggestion
The purpose of `capa` is to analyse the capabilities of a potentially malicious application or file. To achieve this, it needs to include portions of the data it is designed to detect as a basis for comparison.
```

File: doc/faq.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+The release version of capa is packaged with embedded rules to detect common malware functionality. These rules contain similar features to malware and may cause alerts.
```
Comment (VascoSch92) [inline doc/faq.md:4]:
```suggestion
The release version of capa comes with embedded rules designed to detect common malware functionality. These rules possess similar features to malware and may trigger alerts.
```

File: doc/faq.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Additionally, Antivirus and Endpoint Detection and Response (EDR) products may alert on the way capa is packaged using PyInstaller.
+
+## How can I ensure that capa is a benign program?
+We recommedn you only download releases from this repository's Release page. Alternatively, you can build capa yourself or use the other Python installation methods. This project is open source so that everyone can be certain of the project's transparency.
```
Comment (VascoSch92) [inline doc/faq.md:8]:
```suggestion
We recommend downloading releases only from this repository's Release page. Alternatively, you can build capa yourself or use other Python installation methods. This project is open-source, ensuring transparency for everyone involved.
```

File: doc/faq.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+For additinal peace of mind you can use VirusTotal to analyze unknown files against many Antivirus products, sandboxes, and other analysis tools (capa itself runs in VirusTotal).
```
Comment (VascoSch92) [inline doc/faq.md:9]:
```suggestion
For additional peace of mind, you can utilize VirusTotal to analyze unknown files against numerous antivirus products, sandboxes, and other analysis tools. It's worth noting that capa itself operates within VirusTotal.
```

**Review by VascoSch92 — COMMENTED** [review]
Just some typo :-)

**Review by williballenthin — APPROVED** [review]
see also suggestions by @VascoSch92 that are good

**Comment by mr-tz** [comment]
Wow, thanks for all the fixes 😊

**Pull request merged**